test/inductor/test_aot_inductor_package.py

```diff
@@ -176,6 +180,60 @@ def forward(self, x, y):
         )
         self.check_model(Model(), example_inputs)
 
+    def test_compile_after_package(self):
```

Comment:
It would be nice to skip skip tests if `cmake` or `make` could not be found on the tester